src/main/java/org/mariadb/jdbc/internal/util/exceptions/ExceptionMapper.java

```diff
@@ -132,7 +136,7 @@ public static SQLException getException(SQLException exception, MariaDbConnectio
     if (exception.getSQLState() != null) {
       if (connection != null
           && exception.getSQLState() != null
-          && (1205 == exception.getErrorCode() || 1614 == exception.getErrorCode())) {
+          && (IntStream.of(LOCK_DEADLOCK_ERROR_CODES).anyMatch(lockCode -> lockCode == exception.getErrorCode()))) {
```

Comment:
maybe better to use a List definition like 
`private static final List<Integer> LOCK_DEADLOCK_ERROR_CODES = Arrays.asList(1205, 1213, 1614);`
so it can be tested with 
`LOCK_DEADLOCK_ERROR_CODES.contains(exception.getErrorCode())`
avoiding unneeded Stream use

Reply:
Agree
Performance is not the main point here but replaced Stream with a Set, because it's more suitable in terms of contains() usage.